packages/react-avatar/Spec-AvatarGroup.md

```diff
@@ -0,0 +1,130 @@
+# AvatarGroup Spec
+
+## Background
+
+The AvatarGroup component represents multiple people or entities. Avatar groups can be represented in a stacked layout, pie layout, and grid layout.
+
+## Prior Art
+
+### OpenUI
+
+There's no current research in OpenUI's website.
+
+| Library                                                                               | Name         | Notes                                                                                         |
+| ------------------------------------------------------------------------------------- | ------------ | --------------------------------------------------------------------------------------------- |
+| [Fluent UI v8](https://developer.microsoft.com/en-us/fluentui#/controls/web/facepile) | Facepile     | Receives list of `IFacepilePersona` to represent each avatar.                                 |
+| [Attlassian](https://atlassian.design/components/avatar-group/examples)               | Avatar group | Component uses data prop, which is a list of "entries" to represent each avatar in the group. |
+| [Ant Design](https://ant.design/components/avatar/)                                   | Avatar.Group | Uses subcomponent `<Avatar.Group>` to group the avatars given as children.                    |
+| [Primer React](https://primer.style/react/AvatarStack)                                | AvatarStack  | Acts similar to a FlexBox, avatars are given as children and grouped.                         |
+
+### Comparison v8 and v0
+
+There's only one existent component similar to AvatarGroup in v8 `Facepile`. v0 doesn't have an equivalent of this component.
+
+- v8 [Facepile](https://developer.microsoft.com/en-us/fluentui#/controls/web/facepile): Only offers grid layout and offers three overflow indicators.
+
+## Epic issue: [#22240](https://github.com/microsoft/fluentui/issues/22240)
+
+## Sample Code
+
+```jsx
+import { AvatarGroup, Avatar } from '@fluentui/react-avatar';
+
+const App = () => {
+  return (
+    <AvatarGroup layout="grid" size={32}>
+      <Avatar color="colorful" name="Katri Athokas" />
+      <Avatar color="colorful" name="Elvia Atkins" />
+      <Avatar color="colorful" name="Cameron Evans" />
+      <Avatar color="colorful" name="Wanda Howard" />
+      <Avatar color="colorful" name="Mona Kane" />
+    </AvatarGroup>
+  );
+};
+```
+
+## Variants
+
+There are three layout variants in AvatarGroup:
+
+- Grid layout (Default): Avatars are spaced evenly and there can be a maximum of five of them.
+- Stacked layout: Avatars are overlaped evenly and there can be a maximum of five of them.
+- Pie layout: There can be a minimum of two Avatars and a maximum of three. This layout does not overflow and provides a popover for more details.
+
+## API
+
+See [AvatarGroup.types.ts](./src/components/AvatarGroup/AvatarGroup.types.ts) for more details.
+
+## Structure
+
+- _**Public**_
+
+```jsx
+<AvatarGroup layout="grid" size={32}>
```

Comment:
Can you describe how size and individual Avatar sizes are supposed to work? Can you have mixed sizes? Does tha group size override local sizes?